scripts/_Team_Defines.bash

```diff
@@ -71,6 +62,38 @@ alias setup-repository-ci=$RosTeamWS_FRAMEWORK_SCRIPTS_PATH/setup-ci-config.bash
 
 alias setup-ros-workspace=$RosTeamWS_FRAMEWORK_SCRIPTS_PATH/setup-ros-workspace.bash
 
+setup-ros-workspace () {
+    . "$RosTeamWS_FRAMEWORK_SCRIPTS_PATH"/setup-ros-workspace.bash create_workspace "$@"
+}
+
+setup-ros-workspace-docker () {
+   . "$RosTeamWS_FRAMEWORK_SCRIPTS_PATH"/setup-ros-workspace.bash create_workspace_docker "$@"
+}
+
+rtw_switch_to_docker () {
+  if [ -z "$RosTeamWS_WS_DOCKER_SUPPORT" ] || [ "$RosTeamWS_WS_DOCKER_SUPPORT" == false ]; then
+    print_and_exit "It seems your current workspace does not support docker. If it should, did you activate it by executing _\"<ws_alias>\"?"
+  fi
+
+  . "$RosTeamWS_FRAMEWORK_SCRIPTS_PATH"/docker/_RosTeamWs_Docker_Defines.bash start_and_connect_user_to_docker "$RosTeamWS_DOCKER_TAG"
```

Comment:
why not "source ..."?

IMO it is clearer then having just a "."

Reply:
Yes you are right. Done